build/scripts-2.7/hollow

```diff
@@ -0,0 +1,100 @@
+#!/nfs/homes4/chenou/anaconda3/envs/Born/bin/python
+# Hollow --- http://hollow.sourceforge.net
+# Copyright (c) 2009 Bosco Ho & Franz Gruswitz
+
+from hollow import __version__
+from hollow.core import make_hollow_spheres, defaults, help
```

Comment:
I don't think the scripts under build should ever be touched, instead modify the scripts under `scripts`, which are installed.

https://github.com/Becksteinlab/hollow/blob/Interiror-Grid-Fix/scripts/hollow